src/Symfony/Component/PropertyAccess/PropertyAccess.php

```diff
@@ -21,21 +21,35 @@
     /**
      * Creates a property accessor with the default configuration.
      *
+     * @param bool $throwExceptionOnInvalidIndex
+     *
      * @return PropertyAccessor The new property accessor
      */
-    public static function createPropertyAccessor()
+    public static function createPropertyAccessor($throwExceptionOnInvalidIndex = false, $magicCall = false)
     {
-        return self::createPropertyAccessorBuilder()->getPropertyAccessor();
+        return self::createPropertyAccessorBuilder($throwExceptionOnInvalidIndex, $magicCall)->getPropertyAccessor();
     }
 
     /**
      * Creates a property accessor builder.
      *
+     * @param bool $enableExceptionOnInvalidIndex
+     *
      * @return PropertyAccessorBuilder The new property accessor builder
      */
-    public static function createPropertyAccessorBuilder()
+    public static function createPropertyAccessorBuilder($enableExceptionOnInvalidIndex = false, $enableMagicCall = false)
```

Comment:
I don't agree with this change. A builder is exactly there to dynamically set the behavior and not have a huge list of arguments. If there are new flags for the property accessor, you don't want to add every new field as argument for the builder. Then there is no point of a builder.